packages/playground/vue/vite.config.ts

```diff
@@ -2,6 +2,8 @@ import path from 'path'
 import { defineConfig } from 'vite'
 import vuePlugin from '@vitejs/plugin-vue'
 import { vueI18nPlugin } from './CustomBlockPlugin'
+import { createMoveToVendorChunkFn } from 'vite'
+const viteChunks = createMoveToVendorChunkFn()
```

Comment:
The cache is not going to be cleaned up correctly during build watch if it is defined in this way. I think we are having an issue in Vite here, no? This cache should be cleaned in `buildStart`. Maybe we should move it out of the function, and into the plugin control.
Maybe we could end up exposing a single function instead of factory afterwards?

Reply:
Re-analyzed the building logic, the problem does exist, I will push a new commit for review : )

Reply:
Please focus on checking whether it needs to be cleared here?

https://github.com/vitejs/vite/pull/5585/commits/b9f18267465230655699c1c0c0ca82b700d3d9aa#diff-aa53520bfd53e6c24220c44494457cc66370fd2bee513c15f9be7eb537a363e7R564-R565

Will the `parallelCallCounts` introduced by 4a955bcb actually be greater than 1? I tested several scenarios that I expected and found nothing. If it is greater than, maybe parallel build will affect the shared cache?